x/auth/legacy/v0_36/migrate.go

```diff
@@ -0,0 +1,14 @@
+// DONTCOVER
+// nolint
+package v0_36
+
+import (
+	v034auth "github.com/cosmos/cosmos-sdk/x/auth/legacy/v0_34"
+)
+
+// Migrate accepts exported genesis state from v0.34 and migrates it to v0.36
+// genesis state. All entries are identical except for validator slashing events
+// which now include the period.
+func Migrate(oldGenState v034auth.GenesisState) GenesisState {
```

Comment:
This should have a test case:

**Given** `oldGenState` as input
**When** `Migrate(oldGenState)` is called
**Then** output == expected

Reply:
I just added a basic test, I haven't tested the no-op, in this case Params is just forwarded, I was thinking to apply the same logic all over the migrate tests